database/localdb/localdb.go

```diff
@@ -0,0 +1,65 @@
+package localdb
```

Comment:
It would be good to have some documentation (even something basic) on all public symbols and at the package level.